src/app/routes/connector-ui/asset-page/asset-create-dialog/asset-create-dialog.component.ts

```diff
@@ -39,13 +39,13 @@ export class AssetCreateDialogComponent implements OnDestroy {
 
   onSave() {
     const formValue = this.form.value;
-    const assetEntry =
+    const uiAssetCreateRequest =
```

Comment:
same here, I think calling it "assetCreateRequest" is much more helpful than having this emphasis on the "ui" prefix